dagshub/streaming/filesystem.py

```diff
@@ -153,17 +153,23 @@ def _current_revision(self) -> str:
         if self.user_specified_branch:
             branch = self.user_specified_branch
         else:
-            with open(self.project_root / ".git/HEAD") as head_file:
-                head = head_file.readline().strip()
-            if head.startswith("ref"):
-                branch = head.split("/")[-1]
-            else:
-                # contents of HEAD is the revision - check that this commit exists on remote
-                if self.is_commit_on_remote(head):
-                    return head
+            try:
+                with open(self.project_root / ".git/HEAD") as head_file:
+                    head = head_file.readline().strip()
+                if head.startswith("ref"):
+                    branch = head.split("/")[-1]
                 else:
-                    raise RuntimeError(f"Current HEAD ({head}) doesn't exist on the remote. "
-                                       f"Please push your changes to the remote or checkout a tracked branch.")
+                    # contents of HEAD is the revision - check that this commit exists on remote
+                    if self.is_commit_on_remote(head):
+                        return head
+                    else:
+                        raise RuntimeError(f"Current HEAD ({head}) doesn't exist on the remote. "
+                                           f"Please push your changes to the remote or checkout a tracked branch.")
+            except FileNotFoundError:
+                logger.warn("No branch was specified, fetching default branch...")
```

Comment:
```suggestion
                logger.debug("Couldn't get branch info from git repository, fetching default branch from the remote...")
```

Debug because I don't see this being relevant to the user that much